docs/messaging/messaging-spans.md

```diff
@@ -359,92 +397,85 @@ All attributes that are specific for a messaging system SHOULD be populated in `
 
 ### Topic with multiple consumers
 
-Given is a process P, that publishes a message to a topic T on messaging system MS, and two processes CA and CB, which both receive the message and process it.
-
-```
-Process P:  | Span Prod1 |
---
-Process CA:              | Span CA1 |
---
-Process CB:                 | Span CB1 |
+Given is a process P, that publishes a message to a topic T on messaging system MS, and two processes CA and CB, which both receive the message.
```

Comment:
The text now does not align anymore with the new diagram (there's no CA/CB in the mermaid). Would be good if they match, otherwise might be confusing to folks.

Reply:
Fixed.